instaclustr/resource_cluster.go

```diff
@@ -23,6 +23,7 @@ var (
 		"RUNNING":     true,
 		"PROVISIONED": true,
 	}
+	semanticVersioningPattern, _ = regexp.Compile("([0-9]*\\.){2}[0-9]")
```

Comment:
Looks like you're forcing the last digit to be single only, and allowing '.'s with no numbers.
How about ([0-9]+\\.){2}[0-9]+ instead?

Reply:
Ah this is the old regex. I'll change it to the updated version of the regex: [0-9]*\\.[0-9]*(\\.[0-9])?